### ISSUE TITLE
Duplicate relations

### ISSUE THREAD

**Issue by marcverhagen** [marcverhagen/corelex#1]
When creating the html files for relations between duplicate types we get some relations we should not get:

1. Under agt-fod, Rob_Roy.13.0 is a substance holonym of the scotch.13.0 synset and we have the reverse substance meronym relations. But in the browser Rob_Roy.13.0 ends up being both a holonym and meronym of scotch.13.0

2. Under agt-rel, the ampicillin.06.0 synset has a *Domain of synset - USAGE* relation to
trade_name.10.0. The problem is that this relation is listed three times.

**Comment by marcverhagen** [comment]
The second issue is related to the simple way  the source is parsed. The `data.noun` file of WordNet 3.1 has the following line (spread out over several lines for clarity):

```
02708510 06 n 04 ampicillin 0 Principen 0 Polycillin 0 SK-Ampicillin 0 007 
@ 03916152 n 0000 
;u 06864792 n 0401 ;u 06858649 n 0401 ;u 06864792 n 0301 
;u 06858649 n 0301 ;u 06864792 n 0201 ;u 06858649 n 0201 
| semisynthetic penicillin (trade names Principen and Polycillin and SK-Ampicillin)  
```

This includes

```
;u 06864792 n 0401
;u 06864792 n 0301 
;u 06864792 n 0201
```

The fourth field of the pointer refers to the source and target and is `0000` for semantic links. In the example above we have lexical pointers and the relation is between the 2nd, 3rd and 4th element of the source and the 1st element of the target.

Probably the best way to deal with this is to ignore non-semantic links.

**Comment by marcverhagen** [comment]
Fixed with commits bfe0ca2 and db3f068.
